webpack.config.js

```diff
@@ -1,40 +1,36 @@
 "use strict";
 const webpack = require('webpack');
+const TerserPlugin = require("terser-webpack-plugin");
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
```

Comment:
Should this also be commented?

Reply:
nah, this doesn't harm anything to load at build time, it does not get bundled in any case.